client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add.component.js

```diff
@@ -58,20 +58,27 @@ angular.module("managerApp").component("telecomTelephonyAliasMembersAdd", {
         };
 
         self.addMembers = function (form) {
-            self.loaders.adding = true;
-
-            return self.api.addMembers(_(self.addMemberForm.numbers).filter(function (number) {
-                return number && number.length;
-            }).map(function (number) {
-                return _.assign({ number: number }, self.addMemberForm.options);
-            }).value()).then(function () {
-                Toast.success($translate.instant("telephony_alias_members_add_success"));
-                self.resetMemberAddForm();
-                form.$setPristine();
-            }).catch(function (err) {
-                return new ToastError(err);
-            }).finally(function () {
-                self.loaders.adding = false;
+            const modal = $uibModal.open({
```

Comment:
Be aware of mix between es5 and es6 in files, it may not be accepted currently

Reply:
Err, we are using it like this, I just duplicated the previous code used :/

Reply:
OK :)

Reply:
+1 @JeremyDec I definitely prefer to stick with files that just using ES5 or ES6. :candy: